### PR TITLE
style fixes website

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
PR for getting the figma designs in mobile screens

screenshots:
before:
image
image
image
image

after:
image
image
image
image

Signed-off-by: Rakesh PR <rakesh.pr@mayadata.io>

### CODE REVIEW

File: website/src/components/EventSlider/index.tsx [hunk 1]
```diff
@@ -72,6 +72,7 @@ const EventSlider: React.FC<EventsProps> = ({
         settings: {
           slidesToShow: 1,
           centerMode: true,
+          centerPadding: '30px',
```
Comment (Pallavi-PH) [inline website/src/components/EventSlider/index.tsx:75]: can we use theme.spacing here?

File: website/src/components/MiniBlog/index.tsx [hunk 2]
```diff
@@ -143,7 +143,9 @@ const MiniBlog: React.FC = () => {
         breakpoint: mobileBreakpoint,
         settings: {
           slidesToShow: 1,
-          slidesToScroll: 1,
+          arrows: false,
+          centerMode: true,
+          centerPadding: "30px",
```
Comment (Pallavi-PH) [inline website/src/components/MiniBlog/index.tsx:148]: can we use theme.spacing here too?

File: website/src/components/EventSlider/index.tsx (hunk 1 above)
Comment (rakeshPRaghu) [reply website/src/components/EventSlider/index.tsx:75]: As this won't come inside the `makeStyles` the `theme.spacing` will require adding another import of `useTheme` and value will be printed without unit
ex: theme.spacing(3.8) = 30.4, no units will be added.

I think its better if we add this directly @Pallavi-PH

File: website/src/components/EventSlider/style.ts [hunk 3]
```diff
@@ -79,16 +82,29 @@ const useStyles = makeStyles((theme) => ({
       [theme.breakpoints.down("xs")]: {
         padding: `${theme.spacing(0)} !important`
       }
+    },
+    '& .slick-center':{
+      '& $actionLInk' : {
```
Comment (AVRahul) [inline website/src/components/EventSlider/style.ts:87]:
```suggestion
      '& $actionLink' : {
```

File: website/src/components/EventSlider/style.ts [hunk 4]
```diff
@@ -79,16 +82,29 @@ const useStyles = makeStyles((theme) => ({
...
+        }
+      }
     }
   },
   titleText: {
     fontSize: "22px",
     fontWeight: 700,
     color: theme.palette.primary.main,
+    [theme.breakpoints.down("sm")]: {
+      fontSize: 16
```
Comment (IsAmrish) [inline website/src/components/EventSlider/style.ts:99]: please add unit

File: website/src/components/EventSlider/style.ts [hunk 5]
```diff
@@ -79,16 +82,29 @@ const useStyles = makeStyles((theme) => ({
...
+    }
   },
   subText: {
     fontSize: "16px",
     color: theme.palette.primary.dark,
+    [theme.breakpoints.down("sm")]: {
+      fontSize: 14
```
Comment (IsAmrish) [inline website/src/components/EventSlider/style.ts:106]: please add unit

File: website/src/components/MiniBlog/styles.ts [hunk 6]
```diff
@@ -95,6 +109,9 @@ const useStyles = makeStyles((theme: Theme) => ({
   title: {
     fontSize: 22,
     fontWeight: 700,
+    [theme.breakpoints.down("sm")]: {
+      fontSize: 16
```
Comment (IsAmrish) [inline website/src/components/MiniBlog/styles.ts:113]: please add unit

File: website/src/components/MiniBlog/styles.ts [hunk 7]
```diff
@@ -95,6 +109,9 @@ const useStyles = makeStyles((theme: Theme) => ({
   title: {
     fontSize: 22,
```
Comment (IsAmrish) [inline website/src/components/MiniBlog/styles.ts:110]: please add unit

File: website/src/components/MiniBlog/styles.ts [hunk 8]
```diff
@@ -148,6 +165,14 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginRight: theme.spacing(1),
     cursor: "pointer",
   },
+  sliderWrapper: {
+    [theme.breakpoints.down("sm")]: {
+      padding: theme.spacing(0, 1),
+      '& .slick-list': {
+        padding: `${theme.spacing(0, 3,0 ,0)} !important`
```
Comment (IsAmrish) [inline website/src/components/MiniBlog/styles.ts:172]: cannot we avoid `!important`

Comment (rakeshPRaghu) [reply website/src/components/MiniBlog/styles.ts:172]: no the this added to override the inline style added by the script

File: website/src/pages/Home/styles.ts [hunk 9]
```diff
@@ -429,7 +429,10 @@ const useStyles = makeStyles((theme) => ({
         fontWeight: 400,
         color: theme.palette.text.secondary,
         marginTop: theme.spacing(2),
-        wordBreak: 'break-word'
+        wordBreak: 'break-word',
+        [theme.breakpoints.down('sm')]: {
+            fontSize: 14
```
Comment (IsAmrish) [inline website/src/pages/Home/styles.ts:434]: please add unit here as well :)

**Review by AVRahul — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]
LGTM

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**